README.md

```diff
@@ -20,3 +14,52 @@ To run this application, you need to first clone or download the ‘Create a Too
 
 For more help, check the [ASP.NET Core Blazor documentation](https://docs.microsoft.com/en-us/aspnet/core/blazor).
 
+## Features and Benefits
+
+## Offsets
+
+Blazor Tooltip has options to customize the offset position of the Tooltip from the target element. This offset value is defined in pixels.
+
+## Interactivity
+
+By default, the Blazor Tooltip will show and hide when hovering in and out. You can stop the show and hide triggers and make the Tooltip visible always.
+
+## Triggers
+
+You can show tooltips when you hover, click, touch, and focus on any element.
+
+## Easy integration
+
+You just need message and target element to make the tooltip fully functional. It will work out of the box. However, it provides many options to change the position, animation, and appearance.
+
+## Related links
+
+[Learn More about Blazor Tooltip](https://www.syncfusion.com/blazor-components/blazor-tooltip?utm_source=github&utm_medium=listing&utm_campaign=blazor-tooltip-github-samples)
+
+[Download Free Trial](https://www.syncfusion.com/downloads/blazor?utm_source=github&utm_medium=listing&utm_campaign=blazor-tooltip-github-samples)
+
+[Pricing](https://www.syncfusion.com/sales/products/blazor?utm_source=github&utm_medium=listing&utm_campaign=blazor-tooltip-github-samples)
+
+[Documentation](https://blazor.syncfusion.com/documentation/tooltip/getting-started?utm_source=github&utm_medium=listing&utm_campaign=blazor-tooltip-github-samples)
+
+[View Demos](https://github.com/SyncfusionExamples/Create-a-Tooltip-Component-in-a-Blazor-Server-Application?utm_source=github&utm_medium=listing&utm_campaign=blazor-tooltip-github-samples)
+
+[Community Forums](https://www.syncfusion.com/forums/blazor-components?utm_source=github&utm_medium=listing&utm_campaign=blazor-tooltip-github-samples)
+
+[Suggest a feature or report a bug](https://www.syncfusion.com/feedback/blazor-components?utm_source=github&utm_medium=listing&utm_campaign=blazor-tooltip-github-samples)
+
+[Online example](https://blazor.syncfusion.com/demos/toolbar/default-functionalities?utm_source=github&utm_medium=listing&utm_campaign=blazor-tooltip-github-samples)
+
+[Video Tutorials](https://www.syncfusion.com/tutorial-videos/blazor/toolbar?utm_source=github&utm_medium=listing&utm_campaign=blazor-tooltip-github-samples)
```

Comment:
check the demo and video links 
![image](https://user-images.githubusercontent.com/61148953/175478193-bb1221cd-70e3-4782-bb85-a853a753412d.png)

Reply:
control name changed